src/multiCommand.ts

```diff
@@ -9,11 +9,14 @@ export class MultiCommand {
         readonly sequence: Array<Command>
     ) {}
 
-    public async execute() {
+    public async execute(): Promise<Thenable<unknown> | unknown> {
+        let lastOutput: Thenable<unknown> | undefined;
         for (let command of this.sequence) {
-            await command.execute();
+            lastOutput = command.execute();
```

Comment:
Does removing `await` affect the order of execution of the sequence ?
Does the later command execute after the prior command has finished and waited the interval in setting ?